lms/static/sass/discussion/_discussion.scss

```diff
@@ -21,9 +21,8 @@ body.discussion {
 
```

Comment:
I noticed that this Sass partial that is deprecated doesn't have any notes/comments alerting folks to that. Can you add something to the top of this file noting that?